src/picongpu/include/particles/bremsstrahlung/ScaledSpectrum.hpp

```diff
@@ -0,0 +1,312 @@
+            return float_X(0.0);
+        const float_X binKappa = kappa * static_cast<float_X>(electron::NUM_SAMPLES_KAPPA - 1);
+
+        if (picLog::log_level & picLog::CRITICAL::lvl)
+        {
+            if(binE > float_X(electron::NUM_SAMPLES_EKIN - 1))
+            {
+                const float_64 Ekin_SI = Ekin * UNIT_ENERGY;
+                printf("[Bremsstrahlung] error lookup table: Ekin=%g MeV is out of range.\n",
+                       float_X(Ekin_SI * UNITCONV_Joule_to_keV * 1.0e-3));
+            }
+            if(binKappa < float_X(0.0) || binKappa > float_X(electron::NUM_SAMPLES_KAPPA - 1))
+                printf("[Bremsstrahlung] error lookup table: kappa=%f is out of range.\n",
+                       kappa);
+        }
+
+        return this->linInterpCursor[float2_X(binE, binKappa)];
+    }
+};
+
+} // namespace detail
+
+
+/** Generates and holds the lookup tables for the scaled differential cross section
+ * and the stopping power.
+ *
+ * scaled differential cross section = electron energy loss times cross section per unit energy
+ *
+ * stopping power = energy loss per unit length
+ *
+ * The lookup tables are generated from the screened Bethe-Heitler cross section. See e.g.:
+ * Salvat, F., et al. "Monte Carlo simulation of bremsstrahlung emission by electrons."
+ * Radiation Physics and Chemistry 75.10 (2006): 1201-1219.
+ */
+struct ScaledSpectrum
+{
+public:
+    typedef detail::LookupTableFunctor LookupTableFunctor;
+private:
+
+    typedef boost::shared_ptr<PMacc::container::DeviceBuffer<float_X, DIM2> > MyBuf;
+    MyBuf dBufScaledSpectrum;
+    MyBuf dBufStoppingPower;
+
+    /** differential cross section: cross section per unit energy
+     *
+     * This is the screened Bethe-Heitler cross section. See e.g.:
+     * Salvat, F., et al. "Monte Carlo simulation of bremsstrahlung emission by electrons."
+     * Radiation Physics and Chemistry 75.10 (2006): 1201-1219.
+     *
+     * @param Ekin kinetic electron energy
+     * @param kappa energy loss normalized to Ekin
+     */
+    float_64 dcs(const float_64 Ekin, const float_64 kappa) const
+    {
+        namespace pam = PMacc::algorithms::math;
+
+        const float_64 bohrRadius = 4.0 * M_PI * EPS0 * HBAR*HBAR / (ELECTRON_MASS * ELECTRON_CHARGE*ELECTRON_CHARGE);
+        const float_64 classicalElRadius = ELECTRON_CHARGE*ELECTRON_CHARGE / (4.0 * M_PI * EPS0 * ELECTRON_MASS * SPEED_OF_LIGHT*SPEED_OF_LIGHT);
+        const float_64 fineStructureConstant = ELECTRON_CHARGE*ELECTRON_CHARGE / (4.0 * M_PI * EPS0 * HBAR * SPEED_OF_LIGHT);
+
+        const float_64 c = SPEED_OF_LIGHT;
+        const float_64 c2 = c*c;
+        const float_64 m_e = ELECTRON_MASS;
+        const float_64 r_e = classicalElRadius;
+        const float_64 alpha = fineStructureConstant;
+
+        const float_64 W = kappa * Ekin;
+        const float_64 eps = W / (Ekin + m_e * c2);
+        const float_64 R = pam::pow(static_cast<float_64>(ion::TARGET_Z), -1.0/3.0) * bohrRadius;
```

Comment:
@Heikman Are you sure about this equation? In Salvat et al. below Eq. 6, this is just a scaling relation not an equation.